apps/photos/app/globals.css

```
@@ -49,7 +49,10 @@
   --destructive: oklch(0.577 0.245 27.325);
   --destructive-foreground: oklch(0.97 0.014 254.604);
 
-  --ring: oklch(0.623 0.214 259.815);
+  --ring: oklch(70.5% 0.213 47.604);
+  font-family:
+    Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
+    "Segoe UI", sans-serif;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)